lib/Service/ColumnService.php

```diff
@@ -72,51 +77,58 @@ public function find(int $id): Column {
 	/**
 	 * @noinspection PhpUndefinedMethodInspection
 	 * @noinspection DuplicatedCode
-	 * @param int $tableId
-	 * @param string $title
+	 *
 	 * @param string $userId
+	 * @param int $tableId
```

Comment:
PHPdoc comments could be removed here as the type is already defined in the method signature